src/engraving/libmscore/excerpt.cpp

```diff
@@ -1229,6 +1229,12 @@ void Excerpt::cloneStaff(Staff* srcStaff, Staff* dstStaff, bool cloneSpanners)
                             score->undoAddElement(ne1);
                         }
                     }
+
+                    ChordRest* neCr = toChordRest(ne);
+                    for (EngravingItem* crItem : neCr->el()) {
+                        crItem->setTrack(dstTrack);
```

Comment:
I think a different fix is preferable: for ChordRest and Chord, this happens inside their `setTrack` method. See `ChordRest::setTrack` and `Chord::setTrack`. So it is probably better to update those methods (or maybe the `processSiblings` method of those classes instead)

Reply:
@cbjeukendrup I added proccessing of _el in ChordRest, not sure if that's expected, but it works for the task as well